_documentation/video/ofVideoGrabber.markdown

```diff
@@ -67,14 +75,14 @@ Closes the sequence grabber and de-allocates any allocated resources. Call this
 
 <!----------------------------------------------------------------------------->
 
-###void draw(x, y, w, h)
+###void draw(x, y)
 
```

Comment:
the signature of the function and the metadata. everything in between:

```
###void draw(x, y)
<!--
...
-->
```

can't be edited manually cause it's generated by a script whenever we do a release, if you can remove the modifications to this parts, everything else looks great.

In general you should only touch the description.

thanks!